tests/v2/validation/provisioning/permutations/permutations.go

```diff
@@ -107,6 +120,30 @@ func RunTestPermutations(s *suite.Suite, testNamePrefix string, client *rancher.
 
 						provisioning.VerifyRKE1Cluster(s.T(), client, testClusterConfig, clusterObject)
 
+						if strings.Contains(testClusterConfig.CloudProvider, provisioninginput.AWSProviderName.String()) {
+							if strings.Contains(testClusterConfig.CloudProvider, externalProviderString) {
+								err = createAndInstallAWSExternalCharts(client, clusterObject.ID)
+								require.NoError(s.T(), err)
+
+								podErrors := pods.StatusPods(client, clusterObject.ID)
+								require.Empty(s.T(), podErrors)
+							}
+
+							adminClient, err := rancher.NewClient(client.RancherConfig.AdminToken, client.Session)
+							require.NoError(s.T(), err)
+
+							steveClusterObject, err := adminClient.Steve.SteveType(clusters.ProvisioningSteveResourceType).ByID(provisioninginput.Namespace + "/" + clusterObject.ID)
+							require.NoError(s.T(), err)
+
+							lbServiceResp := createAWSWorkloadAndServices(s.T(), client, steveClusterObject)
+
+							status := &provv1.ClusterStatus{}
+							err = steveV1.ConvertToK8sType(steveClusterObject.Status, status)
+							require.NoError(s.T(), err)
+
+							services.VerifyAWSLoadBalancer(s.T(), client, lbServiceResp, status.ClusterName)
```

Comment:
there are a few reasons this should be part of provisioning, mostly to do with parity:
* fundamentally, whether in-tree or out-of-tree, cloud provider setting is meant to enable additional features in the cluster by giving access to 3rd party clients in order to use loadbalancers, upstream storage, etc. Cloud provider is a provisioning setting when creating/updating a cluster
* out of tree cloud provider is the successor to in-tree cloud provider. When In-tree cloud provider is set on the cluster, whether as part of an update to the cluster or a fresh install, apps are automatically added to the cluster. For out-of-tree, the cluster is not usable just by (rancher taints your nodes to wait for the chart to be installed, then the chart removes these taints after its install is successful). so basically the cluster won't work until the charts are installed when selecting out-of-tree, because this setting requires the charts to be installed in order for the cluster to fully 'provision'. The tainted nodes don't allow provisioning to finish until the chart is installed (i.e. rancher-webhook won't install with the taints set)
* rke2 requires that the charts are added, via additional manifest, to the cluster during provisioning, which makes it more clear that this is all to test or enable provisioning features. in RKE2, you can't manually install the charts outside of the create/update cluster workflow. rke1 is a little more bare-bones and you have to do it step by step. 

hopefully this helps clarify why it is provisioning, not a charts test.